airflow/ui/src/components/DurationChart.tsx

```diff
@@ -53,14 +54,16 @@ const average = (ctx: PartialEventContext, index: number) => {
   return values === undefined ? 0 : values.reduce((initial, next) => initial + next, 0) / values.length;
 };
 
-export const RunDuration = ({
-  runs,
-  totalEntries,
+type RunResponse = DAGRunResponse | TaskInstanceResponse;
+
+export const DurationChart = ({
+  entries,
+  kind,
 }: {
-  readonly runs: Array<DAGRunResponse> | undefined;
-  readonly totalEntries: number;
+  readonly entries: Array<RunResponse> | undefined;
+  readonly kind: string;
```

Comment:
```suggestion
  readonly kind: "Dag Run" | "Task Instance";
```

Let's be more specific than just string

Reply:
That would catch the bug we have now where we expect "Task Instances" but we're sending "Task Instance"

Reply:
Thanks, done.